content/pt-br/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md

```diff
@@ -0,0 +1,138 @@
+
+Esta página mostra como usar um Volume para se comunicar entre dois Containers rodando
+no mesmo pod. Veja também como permitir que os processos se comuniquem por
+[compartilhamento do processo de namespace](/docs/tasks/configure-pod-container/share-process-namespace/)
+entre os containers.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+<!-- steps -->
+
+## Criando um pod que executa dois contêineres
+
+Neste exercício, você cria um pod que executa dois contêineres. Os dois recipientes
+compartilham um Volume que eles podem usar para se comunicar. Aqui está o arquivo de configuração
+para a cápsula:
+
+{{< codenew file="pods/two-container-pod.yaml" >}}
+
+No arquivo de configuração, você pode ver que o Pod tem um Volume chamado
+`dados compartilhados`.
+
+O primeiro contêiner listado no arquivo de configuração executa um servidor nginx. o
+o caminho de montagem para o volume compartilhado é `/usr/share/nginx/html`.
+O segundo contêiner é baseado na imagem debian e tem um caminho de montagem de
+`/pod-data`. O segundo contêiner executa o seguinte comando e é encerrado.
+
+    echo Hello from the debian container > /pod-data/index.html
+
+Observe que o segundo contêiner grava o arquivo `index.html` na raiz
+diretório do servidor nginx.
+
+Crie o Pod e os dois Containers:
+
+    kubectl apply -f https://k8s.io/examples/pods/two-container-pod.yaml
+
+Veja informações sobre o Pod e os Containers:
+
+    kubectl get pod two-containers --output=yaml
+
+Aqui está uma parte da saída:
+
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      ...
+      name: two-containers
+      namespace: default
+      ...
+    spec:
+      ...
+      containerStatuses:
+
+      - containerID: docker://c1d8abd1 ...
+        image: debian
+        ...
+        lastState:
+          terminated:
+            ...
+        name: debian-container
+        ...
+
+      - containerID: docker://96c1ff2c5bb ...
+        image: nginx
+        ...
+        name: nginx-container
+        ...
+        state:
+          running:
+        ...
+
+Você pode ver que o contêiner debian foi encerrado e o contêiner nginx ainda está em execução.
+
+Obtenha um shell para o contêiner nginx:
+
+    kubectl exec -it two-containers -c nginx-container -- /bin/bash
+
+Em seu shell, verifique se o nginx está em execução:
+
+    root@two-containers:/# apt-get update
+    root@two-containers:/# apt-get install curl procps
+    root@two-containers:/# ps aux
+
+A saída é semelhante a esta:
+
+    USER       PID  ...  STAT START   TIME COMMAND
+    root         1  ...  Ss   21:12   0:00 nginx: master process nginx -g daemon off;
+
+Lembre-se de que o contêiner debian criou o arquivo `index.html` no diretório raiz do nginx. 
+Use `curl` para enviar uma solicitação GET para o servidor nginx:
+
+```
+root@two-containers:/# curl localhost
+```
+
+A saída mostra que o nginx atende a uma página da web escrita pelo contêiner debian:
```

Comment:
I think "atende" looks a bit out of place here. Some suggestions:
* "serve uma página"
* "responde com uma página"
* "fornece uma página"

Or another option that you feel might suit better.